### DISCUSSION TITLE
Use safe area instes

### DISCUSSION THREAD

**Discussion by zRelux** [nativewind/nativewind#722] [General]
Is there a way to includes in classnames safe area insets?

Would be nice if I could do this:
```
<View className="pt-top" /> // Here it adds the safe area top as a padding top
```

Also using calc on those values would be nice

**Comment by marklawlor** [comment]
Safe area classes were recently added to Tailwind and are on the roadmap

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
